### PR TITLE
CEFI link list - CSV

### PR DESCRIPTION
**Pull request by seventh-hokage**
[cefi_test.csv](https://github.com/user-attachments/files/16462610/cefi_test.csv)

### CODE REVIEW

**Comment by chiaweh2** [comment]
@fifth-hokage thanks! Can you include the .csv file in the directory data/ ?

**Comment by chiaweh2** [comment]
Hi @seventh-hokage if you get a chance please check out the review and change the commit accordingly, Thanks!

**Review by chiaweh2 — COMMENTED** [review]
Please help address the comments! Thanks.

File: data/json_to_csv.txt
[the remark this answers is not in the document]
Comment (seventh-hokage) [reply data/json_to_csv.txt:1]: yes, just added it in case.

File: data/json_to_csv.py
[the remark this answers is not in the document]
Comment (seventh-hokage) [reply data/json_to_csv.py:1]: is data/json_to_csv.pt not currently in the parent directory? or do you mean moving it to the main directory that contains the data/ ?

File: data/json_to_csv.txt
[the remark this answers is not in the document]
Comment (chiaweh2) [reply data/json_to_csv.txt:1]: okay. Please remove this file in this case.

File: data/json_to_csv.py
[the remark this answers is not in the document]
Comment (chiaweh2) [reply data/json_to_csv.py:1]: it is. sorry for the confusion. I meant putting the `json_to_csv.py` directly under the parent directory where other `.py` files are located.

**Comment by chiaweh2** [comment]
incorporated in #137

**Pull request closed without merging**